src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +84,24 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    public String getPrintableString(Printable... printables){
```

Comment:
Consider putting this in the formatter class and use either the StringJoiner or StringBuilder java class for building your string.

Example: https://docs.oracle.com/javase/8/docs/api/java/util/StringJoiner.html